src/sql/workbench/services/objectExplorer/browser/objectExplorerService.ts

```diff
@@ -1004,4 +1019,8 @@ export class ObjectExplorerService implements IObjectExplorerService {
 	public getObjectExplorerTimeout(): number {
 		return this._configurationService.getValue<number>(NODE_EXPANSION_CONFIG);
 	}
+
+	private getTreeNodeCacheKey(node: azdata.NodeInfo): string {
+		return JSON.stringify(node);
```

Comment:
Stringifying the whole node seems like a bit much. Why not just use the path? That should be unique per node.